README.md

```diff
@@ -166,14 +302,14 @@ box.info.lsn
 ...
 ```
 
-#### Lua
+### Lua
```

Comment:
`core = tarantool` and `core = app` tests are really different. It worth to split its description.